app/src/main/java/org/mozilla/fenix/compose/cfr/CFRPopupContent.kt

```diff
@@ -0,0 +1,171 @@
+                .background(
+                    shape = popupShape,
+                    brush = Brush.linearGradient(
+                        colors = listOf(
+                            FirefoxTheme.colors.gradientEnd,
+                            FirefoxTheme.colors.gradientStart
+                        ),
+                        end = Offset(0f, Float.POSITIVE_INFINITY),
+                        start = Offset(Float.POSITIVE_INFINITY, 0f)
+                    )
+                )
+                .wrapContentHeight()
+                .width(popupWidth)
+        ) {
+
+            Column(
+                modifier = Modifier
+                    .padding(
+                        start = 16.dp,
+                        top = 16.dp + if (indicatorDirection == CFRPopup.IndicatorDirection.UP) {
+                            CFRPopup.DEFAULT_INDICATOR_HEIGHT.dp
+                        } else {
+                            0.dp
+                        },
+                        end = 16.dp,
+                        bottom = 16.dp +
+                            if (indicatorDirection == CFRPopup.IndicatorDirection.DOWN) {
+                                CFRPopup.DEFAULT_INDICATOR_HEIGHT.dp
+                            } else {
+                                0.dp
+                            }
+                    )
+            ) {
+                Text(
+                    text = text,
+                    modifier = Modifier.padding(
+                        end = 24.dp, // 8.dp extra padding to the "X" icon
+                    ),
+                    color = FirefoxTheme.colors.textOnColorPrimary,
+                    style = FirefoxTheme.typography.body2
+                )
+
+                action()
+            }
+        }
+
+        IconButton(
+            onClick = { onDismiss(true) },
+            modifier = Modifier
+                .align(Alignment.TopEnd)
+                .padding(
+                    top = if (indicatorDirection == CFRPopup.IndicatorDirection.UP) 14.dp else 0.dp,
+                    end = 6.dp
+                )
+                .size(48.dp)
+        ) {
+            Icon(
+                imageVector = Filled.Close,
+                contentDescription = stringResource(R.string.cfr_dismiss_button_default_content_description),
+                modifier = Modifier
+                    // Following alignment and padding are necessary to visually align the middle
+                    // of the "X" button with the top of the text.
+                    .align(Alignment.TopCenter)
+                    .padding(top = 10.dp)
+                    .size(24.dp),
+                tint = FirefoxTheme.colors.iconOnColor
+            )
+        }
+    }
+}
+
+@Composable
+@Preview(locale = "en", name = "LTR")
+@Preview(locale = "ar", name = "RTL")
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES, name = "Night mode")
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO, name = "Light mode")
```

Comment:
```suggestion
@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES, name = "Dark theme")
@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO, name = "Light theme")
```
I would prefer if we could go with the current theme terminology within Fenix